modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -67,3 +67,10 @@ You can always click on the lock icon next to your file name and unlock it manua
 
 If a user is editing a file and loses their internet connection the lock will timeout, freeing the lock, after 30 minutes. 
 Please refer to {wopi-timeout-documentation-url}[the WOPI documentation] for further information.
+
+== Known Issues
+
+=== Document Locks Are Not Released When Using Google Chrome
+
+When editing a document with Google Chrome (and Chromium) via ownCloud in Office Online, the document lock is _not released_ when the document is closed.
+The document lock is only released after the 30-minute timeout or a manual lock release.
```

Comment:
Note: this is actually a Microsoft bug. So should we mention that? - and that therefore the bug might go away (or get worse ;) in the future out of the control of ownCloud.

Reply:
And could specifically add some thing like:
```
In this case, try to remember to manually unlock the document before closing.
```

Reply:
> Note: this is actually a Microsoft bug. So should we mention that? - and that therefore the bug might go away (or get worse ;) in the future out of the control of ownCloud.

It's definitely worth mentioning, as that makes it clear to the reader that it's not ownCloud's fault nor responsibility. Do you have a link to the issue? If not, I'll search for one.

Reply:
> In this case, try to remember to manually unlock the document before closing.

I'll work that in to this PR.

Reply:
@phil-davis, I've added in two links and [the workaround](https://github.com/owncloud/docs/pull/2474#discussion_r394865058) that you mentioned.